api/package.json

```diff
@@ -30,5 +31,8 @@
   },
   "jest": {
     "testEnvironment": "node"
+  },
+  "script": {
+    "generate-docs": "node_modules/.bin/jsdoc -c jsdoc_config.json"
```

Comment:
curious: is there a difference between `script` and the `scripts` prop being used above? i.e. https://docs.npmjs.com/cli/v7/using-npm/scripts